src/developers/notary_server.md

```diff
@@ -55,7 +55,7 @@ docker run --init -p 127.0.0.1:7047:7047 ghcr.io/tlsnotary/tlsn/notary-server:<v
 ```
 
 ## API Endpoints
-Please refer to the list of all API endpoints [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/tlsnotary/tlsn/main/notary-server/openapi.yaml#/).
+Please refer to the list of all API endpoints [here](./notary_server_api.html).
```

Comment:
```suggestion
Please refer to the list of all HTTP APIs [here](./notary_server_api.html), and WebSocket APIs [here](https://github.com/tlsnotary/tlsn/tree/main/notary-server#websocket-apis).
```

Reply:
Sorry this is not your problem, but haha riding on your PR to make a correction to my past mistake :shh

Reply:
Haha no problem, all good all good 🤫